make transifex-get not working

```
$ make transifex-get
.build/locale/gmf.pot
make: .build/locale/gmf.pot: Command not found
Makefile:687: recipe for target 'transifex-get' failed
make: *** [transifex-get] Error 127
```

--- a/Makefile
+++ b/Makefile
@@ -677,7 +677,7 @@ $(HOME)/.transifexrc:
 	touch $@
 
 .PHONY: transifex-get
-transifex-get: $(L10N_PO_FILES)
+transifex-get: $(L10N_PO_FILES) \
 	.build/locale/gmf.pot
 	# .build/locale/ngeo.pot \
 

